src/dispatch/incident/views.py

```diff
@@ -209,27 +212,69 @@ def update_incident(
 def join_incident(
     *,
     db_session: Session = Depends(get_db),
-    incident_id: str,
+    current_incident: Incident = Depends(get_current_incident),
     current_user: DispatchUser = Depends(get_current_user),
     background_tasks: BackgroundTasks,
 ):
     """
     Join an individual incident.
     """
-    incident = get(db_session=db_session, incident_id=incident_id)
-    if not incident:
-        raise HTTPException(status_code=404, detail="The requested incident does not exist.")
+    background_tasks.add_task(
+        incident_add_or_reactivate_participant_flow,
+        current_user.email,
+        incident_id=current_incident.id,
+    )
+
+
+@router.post(
+    "/{incident_id}/report/tactical",
+    summary="Create a tactical report.",
+    dependencies=[Depends(PermissionsDependency([IncidentEditPermission]))],
+)
+def create_tactical_report(
+    *,
+    db_session: Session = Depends(get_db),
+    tactical_report_in: TacticalReportCreate,
+    current_user: DispatchUser = Depends(get_current_user),
+    current_incident: Incident = Depends(get_current_incident),
+    background_tasks: BackgroundTasks,
+):
+    """
+    Creates a new tactical report.
+    """
+    background_tasks.add_task(
+        report_flows.create_tactical_report(
+            user_email=current_user.email,
+            channel_id=None,
```

Comment:
I think this one is not needed anymore after changing the definition of `create_tactical_report()`.